syntax = "proto3";

//! [0]
package PingService;

message PingMessage {
    string message = 1;
}

message PongMessage {
    string message = 1;
}

service PingPongService {
    rpc Ping(PingMessage) returns (PongMessage);
}
